### ISSUE TITLE
[Suggestion] Projectile Spread

### ISSUE THREAD

**Issue by RacerN2O** [Eniripsa96/SkillAPI#391]
A random spread functionality (provided as max angle) would be really useful for all kinds of projectiles.

**Comment by Eniripsa96** [comment]
Is this for a shotgun sort of effect?

**Comment by RacerN2O** [comment]
Yes, for shotgun like effects with multiple projectiles and for inaccuracy with singular ones as well.
